.github/CONTRIBUTING.rst

```diff
@@ -53,7 +53,8 @@ Talk to us
 ----------
 
 Use Github `discussions`_ forum or for a live chat experience try
-``#ansible-lint`` IRC channel on libera.chat.
+``#ansible-devtools`` IRC channel on libera.chat or Matrix room
+`#devtools:ansible.com <https://matrix.to/#/#devtools:ansible.com>`_.
```

Comment:
Looks like this link is confusing the sphinx `linkcheck`: https://github.com/ansible/ansible-lint/runs/5685992864?check_suite_focus=true#step:7:166

You can probably add a matrix.to regex to `docs/conf.py`. Here are the docs: https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=linkcheck#options-for-the-linkcheck-builder

Reply:
I see. Thank you for the pointers, I'll take a look at it over the next few days then.

Reply:
Not sure if I should use `linkcheck_anchors_ignore` or follow the note in https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=linkcheck#confval-linkcheck_anchors_ignore with `linkcheck_ignore`.

I think it would be preferable to do some checking on the links (not familiar with Sphinx what checks it does exactly, for example checking its not a 404 seems good) and at least not _totally_ ignore those links.